GUI/src/components/HeaderStepCounter/index.tsx

```diff
@@ -68,6 +71,7 @@ const HeaderStepCounter: FC<StepCounterProps> = ({
                 serviceDescription,
                 serviceName,
                 serviceId,
+                isCommon: isCommon,
```

Comment:
You can use the Shorthand Syntax for object property value here as the property name `isCommon` matches the variable name `isCommon`.